recipients_app/test/helpers/golden_test_device_scenario.dart

```diff
@@ -27,7 +27,6 @@ class GoldenTestDeviceScenario extends StatelessWidget {
             size: device.size,
             padding: device.safeArea,
             platformBrightness: device.brightness, 
-            textScaler: TextScaler.linear(device.textScaleFactor),
```

Comment:
Why did you removed this line? Did you run the tests and then it failed and you had to remove this line?

Reply:
No, new version of the dependency just show that there is no more TextScaler 🤔